frontend/assets/stylesheets/_mixins.scss

```diff
@@ -36,6 +36,18 @@ $gs-max-columns: 16;
 @mixin constrain-layout($offset: $gs-gutter * 2) {
     margin: 0 auto;
 
+    @include mq(mobile) {
+        max-width: map-get($max-widths, max-mobile) + $offset;
+    }
+    @include mq(mobile-wider) {
+        max-width: map-get($max-widths, max-mobile-wider) + $offset;
+    }
+    @include mq(mobileLandscape) {
+        max-width: map-get($max-widths, max-mobileLandscape) + $offset;
+    }
+    @include mq(phablet) {
+        max-width: map-get($max-widths, max-phablet) + $offset;
+    }
```

Comment:
I think this will mess with `.l-constrained` layouts in other parts of the site 😬